microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md

```diff
@@ -161,17 +305,56 @@ if (sendMessageId) {
 
 
 <!-- ====================================================================== -->
-## Step 4: Make your `stars.jpeg` available from any browser tab
+## The content script message listener (`content.js`)
 
-This step has already been done in [manifest.json](https://github.com/microsoft/MicrosoftEdge-Extensions/blob/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2/manifest.json) in the MicrosoftEdge-Extensions repo.  If you want to manually create the Part 2 extension, continue with the following steps.
+Here's the `content-scripts\content.js` file that gets injected into every browser tab page based on the `content-scripts` section in `manifest.json`:
 
-To make `images/stars.jpeg` available from any browser tab, you must use the `chrome.runtime.getURL` API.
+`/content-scripts/content.js` (complete listing):
```

Comment:
This code snippet isn't shown anywhere else in the code, so I don't think this line is necessary./ The sentence on line 310 seems like a  sufficient lead-in text.

Reply:
removed line:
``/content-scripts/content.js` (complete listing):`